src/import/FSHImporter.ts

```diff
@@ -352,8 +373,15 @@ export class FSHImporter extends FSHVisitor {
     const valueSet = new FshValueSet(ctx.SEQUENCE().getText())
       .withLocation(this.extractStartStop(ctx))
       .withFile(this.currentFile);
-    this.parseValueSet(valueSet, ctx.vsMetadata(), ctx.vsComponent(), ctx.caretValueRule());
-    this.currentDoc.valueSets.set(valueSet.name, valueSet);
+    if (this.currentDoc.valueSets.has(valueSet.name)) {
+      logger.error(`Skipping value set: a value set named ${valueSet.name} already exists.`, {
```

Comment:
I think we should consider calling this a `ValueSet` in the importer. Same would go for all of the other entities in FSH, I think it makes sense to refer to them in the message by the keyword that the user is seeing. On the export side I could go either way, but on the import side when we are actually referring the the FSH entities, I think the keyword makes sense.